projects/lib/src/dynamic-views/zone/zone.component.ts

```diff
@@ -1,67 +1,111 @@
 import {
-  AfterContentInit,
   ChangeDetectionStrategy,
   ChangeDetectorRef,
   Component,
-  ContentChildren,
-  Input,
-  QueryList,
   OnDestroy,
   OnInit,
   Output,
   EventEmitter,
+  ContentChildren,
+  Input,
+  QueryList,
+  SimpleChanges,
 } from '@angular/core';
-import { Subscription } from 'rxjs';
-import { TemplateNameDirective } from '../../utils/directive/template-name.directive';
+import { BehaviorSubject, Subscription } from 'rxjs';
+
 import { ConfigService } from '../../configuration/config.service';
 import { ConfigurableService } from '../../configurable';
+import { ZoneContextService } from './zone-context.service';
+import { TemplateNameDirective } from '../../utils';
+
 
 @Component({
-  selector: 'uib-zone',
+  selector: 'uib-zone,[uib-zone]',
```

Comment:
Do we really want uib-zone to be used an attribute directive ? This wouldn't be compatible with the make-static directive...

Reply:
We can... but this can be confusing... I remove it